tests/panels/test_sql.py

```diff
@@ -149,7 +149,7 @@ def test_non_ascii_query(self):
         self.assertEqual(len(self.panel._queries), 2)
 
         # non-ASCII bytes parameters
-        list(User.objects.filter(username="café".encode()))
+        list(Binary.objects.filter(field__in=["café".encode()]))
```

Comment:
What's the purpose behind changing this from a `bytes` to a list of `bytes`?

Reply:
I found that even with `BinaryField`, using `.filter(field="café")` would generate a `BytesWarning` internally within Django.  Not wanting to shave that particular yak, I found that switching to the form above allowed the test to proceed without any `BytesWarning`.